spec/bolt/module_installer_spec.rb

```diff
@@ -53,6 +53,15 @@
         /managed by Bolt/
       )
     end
+    it 'errors if module is already in puppetfile without version' do
+      installer.add(new_module, specs, puppetfile, moduledir, project_file, install_config)
```

Comment:
Since this errors with a `File.write` call, I'm not sure how to put `new_module` in the puppetfile without the version automatically being appended. I believe this is the last thing for is.

Reply:
```suggestion
      installer.add(new_module, specs, puppetfile, moduledir, project_file, install_config)
```

You don't want to use the installer here. Instead, you should write to the Puppetfile directly like the test above does on line 48.